package ai.libs.jaicore.experiments.exceptions;

public class ExperimentAlreadyExistsInDatabaseException extends Exception {
	
}
